File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
class Solution {
    public int[] solution(int[] array) {
        int idx = 0;
        int max = -1;
        for(int i = 0; i < array.length; i++){
            if(array[i] > max){
                idx = i;
                max = array[i];
            }
        }
        return new int[]{max, idx};
    }
}

File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.cpp
#include<bits/stdc++.h>

using namespace std;

vector<int> solution(vector<int> array) {
    vector<int> ret;
    ret.push_back(-1);
    ret.push_back(-1);
    for(int i = 0; i < array.size(); i++){
        if(array[i] > ret[0]){
            ret[0] = array[i];
            ret[1] = i;
        }
    }
    return ret;
}
/*


*/